--- iframeResizer.js	2017-03-30 18:41:38.000000000 +1000
+++ iframeResizer.js	2017-09-10 08:38:37.103615634 +1000
@@ -9,7 +9,7 @@
  */
 
 
-;(function(undefined) {
+;(function(window) {
 	'use strict';
 
 	if(typeof window === 'undefined') return; // don't run for server side render
@@ -1037,4 +1037,4 @@
 		window.iFrameResize = window.iFrameResize || factory();
 	}
 
-})();
+})(Function('return this')());
